docs/vmanomaly.md

```diff
@@ -126,43 +126,67 @@ optionally preserving labels).
 
  
 ## Usage
-The vmanomaly accepts only one parameter -- config file path:
+> Starting from v1.5.0 vmanomaly requires a license key to run. You can obtain a trial license key [here](https://victoriametrics.com/products/enterprise/trial/).
 
-```sh
-python3 vmanomaly.py config_zscore.yaml
-```
-or
-```sh
-python3 -m vmanomaly config_zscore.yaml
-```
+> You can find the Getting started with vmanomaly guide [here](./guides/guide-vmanomaly-vmalert.md)
```

Comment:
```suggestion
> See [Getting started guide](https://docs.victoriametrics.com/guides/guide-vmanomaly-vmalert.html).
```
It is better to use absolute links instead of relative.